app/serializers/osf-serializer.ts

```diff
@@ -132,6 +140,17 @@ export default class OsfSerializer extends JSONAPISerializer {
         return underscore(key);
     }
 
+    getGraphRelationships(snapshot: DS.Snapshot) {
+        const graph = peekGraph(this.store);
+        const identifiers = graph.identifiers;
+        for (const [key, value] of identifiers) {
+            if(key.id === snapshot.id && key.type === snapshot.type.modelName) {
+                return value;
```

Comment:
So this is not great. I'm asking on the discord for the proper way to do this. Right now, it'll scan through everything in the ember data store to get the right record. It's not _the worst_, because nobody should have hundreds of thousands of records, and everything's in memory, so it should be pretty quick at the worst of times, but still, there is definitely a better way of getting this info and hopefully I'll find out what that is.